frontend/docsettings.lua

```diff
@@ -165,7 +173,7 @@ function DocSettings:_findCoverFileInDir(dir)
     local ok, iter, dir_obj = pcall(lfs.dir, dir)
     if ok then
         for f in iter, dir_obj do
-            for _, ext in ipairs(self.cover_ext) do
+            for ext in pairs(self.image_ext) do
```

Comment:
I think I remember we discussed that when you made it: using pairs() will get non-stable iteration, so if 2 cover files are available, cover.jpg and cover.png, a different one may be picked on different runs. Using ipairs() allows the files to be picked according to the order in cover_ext/image_ext.
Dunno if that's still an issue the way this is used.

Reply:
There shouldn't be two custom covers and it is impossible to set the second one within the program.
The only way is to load the second custom cover out of KOReader. In that case unstable cover will indicate to the user that something is wrong.

Reply:
> The only way is to load the second custom cover out of KOReader. In that case unstable cover will indicate to the user that something is wrong.

Ok, but previously, the user putting 2 cover files in the sdr directory did not lead to "something is wrong" :) We just preferred one to the other, and always the same.
Also, "unstable cover will indicate" is a bit of an unstable indication :) and may have users reporting issues that could lead us to think there is another problem on our side.